### DISCUSSION TITLE
Maybe should have waited...

### DISCUSSION THREAD

**Discussion by FLATFOOTGAMES** [termux/termux-packages#9568] [Q&A]
[?] Delete downloaded rootfs file? [y/N] n

[*] Configuring NetHunter for Termux ...
~ $ nh
proot error: execve("/usr/bin/env"): Function not implemented
proot info: possible causes:
  * the program is a script but its interpreter (eg. /bin/sh) was not found;
  * the program is an ELF but its interpreter (eg. ld-linux.so) was not found;
  * the program is a foreign binary but qemu was not specified;
  * qemu does not work correctly (if specified);
  * the loader was not found or doesn't work.
fatal error: see `proot --help`.
proot error: can't chmod '/data/data/com.termux/files/usr/tmp/proot-21368-36gRVD': No such file or directory how do I go back?

**Comment by Exanoff12** [comment]
what tutorial did you follow ?

looks like reinstallation is the way to go. do `termux-reset` and then try again. also clear out any existing catch file in `~` so something wrong doesn't happen again.
